### PR TITLE
DOCS 2.4 : fix internal links

### PR DESCRIPTION
**Pull request by SpiritLevel**

### CODE REVIEW

**Comment by dhensby** [comment]
@SpiritLevel has this been pulled in somewhere else?

You should try not to delete your repo - are you trying to solve a specific problem every time you do?

**Comment by SpiritLevel** [comment]
@dhensby  Yes, this has been pulled in already. I'll be revisiting these links once my docsviewer PR is merged (which there is a bit more to do on).

I tend to experiment with git and get myself into trouble, at which point I delete everything and start from scratch...I shall try to avoid future deletions!

**Comment by dhensby** [comment]
If you ever need to get a branch back into the state of "upstream" ie: the SS repo, you just need to do: 

`git reset --hard upstream/[branch name]`

Where upstream is the SS repo set like so (`git remote add upstream https://github.com/silverstripe/silverstripe-framework`)

You'll need to run `git fetch upstream` too to make sure it's up to date. 

Note the `reset --hard` will discard all changes in your branch. 

Basically run like so: 

 ```
git checkout 3
git remote add upstream https://github.com/silverstripe/silverstripe-framework
git fetch upstream
git reset --hard upstream/3
```

**Comment by SpiritLevel** [comment]
You're on a marathon more than a sprint! 

Thanks for the git tips :) 

I was experimenting with reverts in my local repo and somehow Damian got associated with my PR! And the log was all messed up. Weird.

See https://github.com/silverstripe/silverstripe-docsviewer/pull/91

I would have done a reset or recloned my github fork but I didn't want to lose all the work I had done...

Anyhow, thanks....and thanks for merging the docsviewer PR. More to come.

**Comment by dhensby** [comment]
If you run `git reflog` you'll see all the changes/commits to your working directory so even if you do a `reset --hard` you can get your commits back
